resources/lang/en/Template.properties

```
@@ -559,6 +559,9 @@ myAccountSave = "Save"
 myAccountSettings = "Account settings"
 myAccountShippingAddresses = "Delivery addresses"
 ; myAccount - My account
+; navigation
+menuLabel = "Menu"
```

```suggestion
menuLabel = "Open mobile navigation menu"
```
I think this is a bit more descriptive